reporting-app/db/migrate/20251210000001_create_ex_parte_activities.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class CreateExParteActivities < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ex_parte_activities, id: :uuid, comment: "Hours data from external sources (API/batch) for compliance calculation" do |t|
+      t.string :member_id, null: false,
+               comment: "Member reference - always required (allows hours-first save order)"
+      t.uuid :certification_id,
+             comment: "Certification reference - nullable for pending entries before cert creation"
+      t.string :category, null: false,
+               comment: "Activity category: employment, community_service, education"
+      t.decimal :hours, precision: 8, scale: 2, null: false,
+                comment: "Hours worked/volunteered (max 744 = 31 days × 24 hours)"
+      t.date :period_start, null: false,
+             comment: "Activity period start date"
+      t.date :period_end, null: false,
+             comment: "Activity period end date"
+      t.boolean :outside_period, default: false, null: false,
+                comment: "True if activity dates are outside certification lookback period"
```

Comment:
don't think we should have this, we should just tie the data to the member not the certification

Reply:
see comment above about it being tied to memeber, but with our app more certification centric UI , we should maintain data about how it related to certs even if record officially is related to member. otherwise we are doing on the fly calcs/queries that slow things down and our data is inferred , not direct.